README.md

```diff
@@ -11,56 +11,85 @@ Swarm Control Planes.
 
 ### AWS
 
-- [11.2.1](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.2.1.md)
-- [11.2.0](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.2.0.md)
-- [11.1.4](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.1.4.md)
-- [11.1.3](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.1.3.md)
-- [11.1.2](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.1.2.md)
-- [11.0.1](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.0.1.md)
-- [11.0.0](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.0.0.md)
-- [10.1.2](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v10.1.2.md)
-- [10.1.1](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v10.1.1.md)
-- [10.1.0](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v10.1.0.md)
-- [9.2.5](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.2.5.md)
-- [9.2.4](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.2.4.md)
-- [9.2.3](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.2.3.md)
-- [9.2.2](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.2.2.md)
-- [9.2.1](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.2.1.md)
-- [9.2.0](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.2.0.md)
-- [9.1.0](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.1.0.md)
-- [9.0.3](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.0.3.md)
-- [9.0.2](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.0.2.md)
-- [9.0.1](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.0.1.md)
-- [9.0.0](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v9.0.0.md)
-- [8.5.0](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v8.5.0.md)
+- v11
+  - v11.2
```

Comment:
Personal opinion:
This makes the page way too busy / noisy. Just whitespace between minors would do the same job without adding loads of redundant information.

Not blocking just stating my preference.

Reply:
Thanks, acknowledged